docker-compose.yaml

```diff
@@ -1,4 +1,4 @@
-version: '3.8'
```

Comment:
Trzeba zostawić numer wersji bo może być problem w zależności to jaką wersję wybierze przy buildzie